geomesa-nifi-processors/src/main/scala/org/geomesa/nifi/geo/AbstractGeoIngestProcessor.scala

```diff
@@ -51,14 +53,8 @@ abstract class AbstractGeoIngestProcessor(dataStoreProperties: Seq[PropertyDescr
   private var descriptors: Seq[PropertyDescriptor] = _
   private var relationships: Set[Relationship] = _
 
-  @volatile
-  private var dataStore: DataStore = _
```

Comment:
I wonder if the DataStore was volatile for some reason?

Reply:
it's wrapped in the Writer now, which is still volatile, so I think it should be equivalent. not sure why it's volatile to begin with - usually that has to do with serialization.